_docs/manage/cluster.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: guide
+title: StorageOS Docs - Cluster
+anchor: manage
+module: manage/cluster
+---
+
+# Cluster overview
+
+A StorageOS cluster needs to know the exact cluster size and peers to connect to
+during start up. All the nodes need to bind to the correct addresses in order to
```

Comment:
This might be confusing until we release 0.8.  With 0.7, we don't need to know the cluster size, only the KV_ADDR.  With 0.8 we need to know the number of KV members at startup, but this is not the same as the cluster size which can grow to a yet unknown size.

Highlightling that you should set the ADVERTISE_IP is a very good idea as this has been causing some problems.

Also, not sure if you want to add it here, but maybe saying `storageos node ls` is a good way to show cluster status?